webpack/private/is_bazel_7_or_greater.bzl

```diff
@@ -0,0 +1,7 @@
+"is_bazel_7_or_greater"
+
+def is_bazel_7_or_greater():
```

Comment:
This file will be removed once on rules_js 2.0 and with a minimum of bazel-lib 2.x

Reply:
Is the principled answer to use `bazel_features` and be conditional for this?

Reply:
Using it for presence of cache.bzl . Nothing in bazel_features for that.